docs/content/concepts/assets/asset-checks/checking-for-data-freshness.mdx

```diff
@@ -356,6 +357,41 @@ height={1944}
 
 ---
 
+## Alert on overdue assets (Dagster+ only)
+
+In Dagster+, you can set up [alerts](/dagster-plus/managing-deployments/alerts) to notify you when an asset is overdue for an update. By default, a freshness check will fail with severity `WARN` if an asset is overdue for an update. Then, in order to alert on overdue assets, you'll want to create an alert policy that triggers on asset check failures with severity `WARN` (or `ERROR` if you've overridden the default severity), for your specific assets which you've defined freshness checks for. Asset alert policies can currently be targeted at specific asset keys, asset groups, or all assets.
+
+To become more familiar with the various ways alert policies can be set up, see the guide on [managing alert policies](/dagster-plus/managing-deployments/alerts/managing-alerts-in-ui).
+
+<TabGroup>
+<TabItem name="Option 1: Use the alert policy editor UI">
```

Comment:
Sorry to continue to nitpick here, but can we adopt the approach [for tabs used here](https://docs.dagster.io/dagster-plus/managing-deployments/alerts/email#creating-an-alert-policy), for consistency?

Reply:
good call